src/main/java/eu/ai4work/sws/service/SlidingDecisionService.java

```diff
@@ -0,0 +1,67 @@
+package eu.ai4work.sws.service;
+
+import eu.ai4work.sws.model.specific.CongestionLevel;
+import eu.ai4work.sws.model.specific.TruckPosition;
+import eu.ai4work.sws.model.generic.SlidingDecisionResult;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Service
+@RequiredArgsConstructor
+public class SlidingDecisionService {
+    private final RuleEngineService ruleEngineService;
+    public Map<String, Object> getSlidingDecision(Map<String, Object> parameters) {
```

Comment:
I think that this should better just return a value of type `SlidingDecisionResult`, while the output `Map` could better be generated in the `RestController` (as that is pure formatting of output, does not have anything to do with the decision logic provided by this service).

Reply:
I agree on Batter to just return the `SlidingDecisionResult` and output `Map` batter to generate in the `RestController`. This changes done in the commit https://github.com/AI4WORK-Project/sliding-work-sharing/pull/2/commits/6e63b8bce37097860fbe069304276275bb28ba83